src/vs/platform/layout/browser/layoutService.ts

```diff
@@ -94,6 +95,13 @@ export interface ILayoutService {
 	 */
 	readonly activeContainerOffset: ILayoutOffsetInfo;
 
+	/**
+	 * A Barrier opened when the stylesheets for the active container have been
+	 * loaded. Aux windows load their styles asynchronously, so this barrier
+	 * may initially be closed.
+	 */
+	readonly activeContainerStylesLoaded: Barrier;
```

Comment:
Rather than exposing the `Barrier`, I would have expected a `whenActiveContainerStylesLoaded: Promise`, similar to other examples in services such as lifecycle service. This also makes sure, no one else opens the `Barrier`